test/test_manifest.json

```diff
@@ -2247,6 +2247,12 @@
        "lastPage": 4,
        "type": "eq"
     },
+    {  "id": "issue4227",
+      "file": "pdfs/coons-allflags-withfunction.pdf",
+      "md5": "c5f79c24bf9eb66698be0e4ecaa1bdf8",
+      "rounds": 1,
+      "type": "eq",
```

Comment:
i can fixup the indention in this file, but which one is the wanted one. give me a reference test case and ill do it now